ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/processed/VariableDataProvider.kt

```diff
@@ -0,0 +1,60 @@
+package com.revenuecat.purchases.ui.revenuecatui.data.processed
+
+import android.content.Context
+import com.revenuecat.purchases.Package
+import java.text.NumberFormat
+import java.util.Currency
+import java.util.Locale
+
+@Suppress("UnusedParameter", "FunctionOnlyReturningConstant")
+internal class VariableDataProvider(
+    private val applicationContext: Context,
+) {
+    companion object {
+        private const val MICRO_MULTIPLIER = 1000000.0
```

Comment:
Is it possible to write this as `1_000_000.0`

Reply:
I've actually removed it since it's been moved to `PriceExtensions` in #1255